### PR TITLE
Add Profile

### PR DESCRIPTION
**Pull request by Vishal0703**
Add Vishal Gupta as a software team member.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -400,6 +400,15 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                   				<span>Member, Software Team</span>                  
                   			</div>
                   		</div>
+                </article>
+                	<article class="col-md-2 isotopeItem software">
```
Comment (pranitbauva1997) [inline index.html]: The indentation isn't really accurate. But the whole file's indentation is a mess so I guess it is okay.

**Comment by pranitbauva1997** [comment]
You have opened the PR from master branch which is wrong because it would hamper the commit history.

Just use a simple logic, since you don't have write access to master branch, don't commit to master branch. When you commit to master branch and then pull, the commit history gets hampered.

To fix this follow these steps:

`$ git checkout -b my-profile`
`$ git push origin my-profile`
`$ git checkout master`
`$ git reset --hard HEAD~1`

And then open this PR again from the branch my-profile. I am closing this as for now!

**Comment by Vishal0703** [comment]
i created another branch named member and the i did the changes on that
branch and i sent pull request via that branch only. I don't know why it
was showing the master branch.

**Comment by pranitbauva1997** [comment]
Did you just use `git branch new-branch` and start working?

If yes, that just creates the branch but doesn't check it out. You would then have to again check it out by using `git checkout new-branch`. Or in just in command, `git checkout -b new-branch`

**Comment by Vishal0703** [comment]
I checked out member branch.

I am sending you the screenshots of git log after checking out both the
branches. You can see there that in master nothing has been changed,
whereas I have committed changes in the member branch.

**Comment by Vishal0703** [comment]
I checked out member branch.
I am sending you the screenshots of git log after checking out both the
branches. You can see there that in master nothing has been changed,
whereas I have committed changes in the member branch.

**Pull request closed without merging**